### ISSUE TITLE
Error when saving chat

### ISSUE THREAD

**Issue by AlessandroZanatta** [mahtoid/DiscordChatExporterPy#34]
I've encountered the following error. This may be related to some of the new features of Discord, but I'm unsure as I've not used this library in a while.
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/chat_exporter/chat_exporter.py", line 36, in export
    return (await Transcript.export(channel, limit, set_timezone)).html
  File "/usr/local/lib/python3.10/site-packages/chat_exporter/chat_exporter.py", line 100, in export
    transcript = await Transcript(channel=channel, guild=channel.guild, messages=messages,
  File "/usr/local/lib/python3.10/site-packages/chat_exporter/chat_exporter.py", line 119, in build_transcript
    message_html += await Message(m, previous_message, self.timezone_string).build_message()
  File "/usr/local/lib/python3.10/site-packages/chat_exporter/chat_exporter.py", line 189, in build_message
    await self.generate_message_divider()
  File "/usr/local/lib/python3.10/site-packages/chat_exporter/chat_exporter.py", line 213, in generate_message_divider
    self.message_html += await fill_out(self.message.guild, start_message, [
  File "/usr/local/lib/python3.10/site-packages/chat_exporter/build_html.py", line 36, in fill_out
    base = base.replace("{{" + k + "}}", v)
TypeError: replace() argument 2 must be str, not MessageReference
Please send a screenshot of the above error to https://www.github.com/mahtoid/DiscordChatExporterPy
```

**Comment by mahtoid** [comment]
Hello @AlessandroZanatta - I'm unable to replicate this issue locally (using the latest dpy, chat-exporter and the different styles of message references). 

When using the latest version of discord.py, make sure you are using the latest version of `chat-exporter`.
You can upgrade with `pip install --upgrade chat-exporter`.

If you have done this and you are still getting an issue - please could you give me a little more information (perhaps to do with the message references ('Reply' feature) within the channel you are trying to export.

**Comment by AlessandroZanatta** [comment]
Sorry for the late answer.

I actually had an old version running on my container, but unfortunately that's not the only problem (I've also seen a couple of other issues have been opened).
Here's the updated error:

![image](https://user-images.githubusercontent.com/41871589/160290883-824dcf55-1f64-49ab-a008-b2f28eecd1a1.png)

I've simply called the export function with the channel name. Regarding the message references, let me know if I still need to dig into that, but I think the problem now is unrelated to that. I was able to reproduce the same exact error with a channel with a single message in it.
